### ISSUE TITLE
Theming icons and texts (and others) on cards

### ISSUE THREAD

**Issue by Tamas-Toth-ebola** [home-assistant/frontend#5418]
## The request

Hello,

I would like to make a theme but got some maybe missing entities in the theming options what I firstly asked in the HA - Community but without answer I'm trying to ask now here:

Please check my post here: https://community.home-assistant.io/t/theming-icons-and-texts-and-others-on-cards/181769

Thanks a lot for your efforts in advance!

**Comment by bramkragten** [comment]
Github issues are not for support.

**Comment by Tamas-Toth-ebola** [comment]
Sorry I thought the dedicated `feature request` are for feature requests. But in this case I will not do again.

**Comment by bramkragten** [comment]
It is for feature requests, not for questions on how to create a theme.
And if you have a real request, fill out the template and make it clear what you want and why and why it can't be done in another way.

**Comment by Tamas-Toth-ebola** [comment]
Ok! I understand as this is why I wrote that first time I asked it from community to not disturb the developers. There I think I detailed my problems enough for understand with proper and strict resolutions and references.

But because I got no answer I thought maybe this could be not possible yet as there are not enough variables for it what is not problem just this si why I wrote here my next try. My only fault I think that I just linked the original details instead of reformatting here redundantly. This is really a mistake.

Anyway sorry, I got your answer that's all.

Thanks,
